### PR TITLE
April 2020 package updates

### PR DESCRIPTION
**Pull request by niall3rs**
### Description

Routine package updates

cc @zendesk/radar 

### Risks

* Medium : Package updates, with a couple of breaking changes
* Rollback: Revert

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,28 +50,28 @@
     "javascript-state-machine": "^3.1.0",
     "lodash": "^4.17.15",
     "miniee": "0.0.5",
-    "minilog": "2.0.8",
-    "mobx": "^5.15.0",
+    "minilog": "^3.1.0",
```
Comment (thomasbui93) [inline package.json:53]: Careful with Minilog update. It could disable logging entirely.

Comment (niall3rs) [reply package.json:53]: ah yeah, I thought I remembered something about that. How did we test it last time?

Comment (thomasbui93) [reply package.json:53]: Last time, I just reverted to the old version when I tested in zendesk_radar.

Comment (niall3rs) [reply package.json]: putting it back to 2 until we have time to validate it doesn't cause issue upstream 👍

**Pull request merged**